.github/workflows/codeql-analysis.yml

```
@@ -19,6 +19,21 @@ on:
     branches: [ master ]
   schedule:
     - cron: '32 2 * * 6'
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: 'Log level'     
+        required: true
+        default: 'warning' 
+        type: choice
+        options:
+        - info
+        - warning
+        - debug 
+      tags:
+        description: 'Test scenario tags'
+        required: false 
+        type: boolean
```

```suggestion
  workflow_dispatch:
```

I don't think we need the tagging as all and the log level doesn't seem to be referenced anywhere.